src/test/java/com/epam/tc/hw5/pages/LeftNavBar.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.tc.hw5.pages;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LeftNavBar {
+
+    private String leftMenuHomeName = "Home";
```

Comment:
Better in enum, believe me :)

Reply:
added enum, refactored LeftNavBar file using enum